projects/sandbox/timeslide_injections/timeslide_injections.py

```diff
@@ -105,6 +105,16 @@ def inject_waveforms(
         # store snr
         parameters[f"{ifo}_snr"] = snr
 
+        # alternatively:
+        idx = np.arange(signals.shape[-1])[None]
+        idx = np.repeat(idx, len(signals), axis=0)
+        time_diffs = signal_times - times[0]
+        idx_diffs = (time_diffs / sample_rate).astype("int64")
+        idx += idx_diffs[:, None]
+        idx = idx.reshape(-1)
+        signals = signals.reshape(-1)
+        x[idx] += signals
+
```

Comment:
Also implemented in #150

Reply:
Oh shoot I'm fairly confident I didn't mean to implement this here, I think it's just where I mocked this idea up and forgot to take it out. I'll checkout this file back to `main`, and we'll use the implementation in #150